website/docs/getting-started/setup/installation-guides/docker/README.mdx

```diff
@@ -1,203 +1,84 @@
 ---
-description: Appsmith can be deployed locally or on a private instance using Docker
-sidebar_position: 1
+description: Deploy Appsmith locally or on a private instance with ease using Docker Compose. Follow these simple steps to set up Appsmith on your Docker Compose platform.
 ---
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
 # Docker
 
-Docker is an open source [containerization](https://www.ibm.com/in-en/cloud/learn/containerization) platform. It enables developers to package applications into standardized executable components called containers. These containers combine the application's source code with the Operating System (OS) libraries and dependencies required to run that code in any environment.
+This page provides stepwise instructions to help you install Appsmith using Docker.
```

Comment:
```suggestion
This page provides instructions for the quickest way to install Appsmith locally using Docker.
```

Reply:
I don't think we should add the quickest way. I'll reword this.